src/hyperion/external_interaction/nexus/nexus_utils.py

```diff
@@ -65,6 +65,57 @@ def create_goniometer_axes(
     return Goniometer(gonio_axes, scan_points)
 
 
+def create_vmxm_goniometer_axes(
+    omega_start: float,
+    scan_points: dict | None,
+    x_y_z_increments: tuple[float, float, float] = (0.0, 0.0, 0.0),
+    chi: float = 0,
+    phi: float = 0,
+) -> Goniometer:
+    """Returns a Nexgen 'Goniometer' object with the dependency chain of I03's Smargon
```

Comment:
```suggestion
    """Returns a Nexgen 'Goniometer' object with the dependency chain of VMXm's axes
```

Comment is obviously wrong!